Robust.Client/UserInterface/Controls/OptionButton.cs

```diff
@@ -14,6 +14,7 @@ public class OptionButton : ContainerButton
         public const string StyleClassOptionButton = "optionButton";
         public const string StyleClassPopup = "optionButtonPopup";
         public const string StyleClassOptionTriangle = "optionTriangle";
+        public const string StyleClassOptionsBackground = "optionsBackground";
```

Comment:
```suggestion
        public const string StyleClassOptionsBackground = "optionButtonBackground";
```

We have a few different controls with Option as part of the name, so it's better to differentiate more just in case.